app/assets/response_1.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Heya, thanks for the review, I'll work through it now. These files are used to test how the responses from the support-api are paginated. Is there a way that mocks are usually handled? The integration addresses are just there because that's where the original data comes from but they're not used in the tests.

Reply:
Usually test fixtures like this go in the `spec/fixtures` or `test/fixtures` directory. You may need to add something in your tests to load them.

Reply:
cool thanks, will move them over